plugin/src/main/java/io/jenkins/plugins/casc/model/Mapping.java

```diff
@@ -9,6 +9,8 @@
 
 public final class Mapping extends HashMap<String, CNode> implements CNode {
 
+    // Deprecated by PR #1240
+    public static final Mapping EMPTY = new Mapping();
```

Comment:
Add the deprecated annotation

Reply:
Done

Reply:
Please use proper deprecation annotation:
https://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html

Reply:
I assume you mean add the java doc deprecation to document what to use instead?

Reply:
Yes :)